server_admin/topics/clients/client-policies.adoc

```diff
@@ -80,6 +80,9 @@ Client Role::
 Client Domain Name, Host or IP Address::
     Applied for specific domain names of client. Or for the cases when the administrator registers/updates client from particular Host or IP Address.
 
+Client Client-IDs::
+    Evaluates to true if a clients client-id is configured in the condition.
```

Comment:
Change "clients" to "client's"
Also to eliminate the Error reports that are expected to finish, but are stuck, issue this command:
git pull upstream main
then commit again